server/@types/viewModels/index.d.ts

```diff
@@ -116,6 +116,7 @@ declare module 'viewModels' {
    */
   export interface WorkAndInterests {
     problemRetrievingData: boolean
+    inductionQuestionSet?: 'SHORT_QUESTION_SET' | 'LONG_QUESTION_SET'
```

Comment:
I wish we could model and export enums in these type interfaces, but it seems we cant 😢